src/Processors/QtiV2/In/Interactions/ChoiceInteractionMapper.php

```diff
@@ -15,14 +18,21 @@
 
 class ChoiceInteractionMapper extends AbstractInteractionMapper
 {
+
     public function getQuestionType()
     {
         /* @var QtiChoiceInteraction $interaction */
         $interaction = $this->interaction;
 
         $options = $this->buildOptions($interaction->getSimpleChoices());
+        $metadata = $this->buildFeedbackMetadata($interaction->getSimpleChoices());
         $mcq = new mcq('mcq', $options);
 
+        // Support for @mcq-metadata
+        $metaData = new mcq_metadata();
```

Comment:
This is ambiguously named as there is already a `$metadata` variable above, which becomes very confusing to read.

Try using more specific names, like `$feedbackMetadata` and `$mcqMetadata`.

Reply:
Done